src/site.webmanifest

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "",
```

Comment:
In order to be valid, the manifest needs to have these fields filled.

![image](https://user-images.githubusercontent.com/20233319/63336812-13dbb580-c340-11e9-84e2-740d722c0516.png)